reference/shaders/vulkan/rgen/shader_record_buffer.nocompat.vk.rgen.vk

```diff
@@ -0,0 +1,17 @@
+#version 460
+#extension GL_NV_ray_tracing : require
+
+layout(shaderRecordNV) buffer sbt
```

Comment:
What about packing qualifiers like std140 and std430, do they apply here?

Reply:
`shaderRecordNV` cannot be used together with any other layout qualifiers (https://github.com/KhronosGroup/GLSL/blob/master/extensions/nv/GLSL_NV_ray_tracing.txt#L465).

Reply:
I see, so it's effectively "logical" storage?

Reply:
You are right, my mistake. It does indeed accept packing qualifiers.